### PR TITLE
some simple tests

### PR DESCRIPTION
**Pull request by CyberDem0n**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+PyYAML
+psycopg2
+requests
```
Comment (valgog) [inline requirements.txt]: From my experience with pydidit, the version of requests can be quite important.

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+__location__ = os.path.join(os.getcwd(), os.path.dirname(inspect.getfile(inspect.currentframe())))
+
+
+NAME = 'governor'
+MAIN_PACKAGE = 'governor.py'
+HELPERS = 'helpers'
+VERSION = '0.1'
+DESCRIPTION = 'A Template for PostgreSQL HA with etcd'
+LICENSE = 'The MIT License'
```
Comment (valgog) [inline setup.py]: Guys, areare you sure the MIT lisence is the right one?

Comment (CyberDem0n) [inline setup.py]: Are we allowed to change it?

File: helpers/postgresql.py [hunk 3]
```diff
@@ -79,12 +79,9 @@ def data_directory_empty(self):
...
     def initialize(self):
-        if os.system(self._pg_ctl + ' initdb -o --encoding=UTF8') == 0:
-            self.write_pg_hba()
-
-            return True
-
-        return False
+        ret = os.system(self._pg_ctl + ' initdb -o --encoding=UTF8') == 0
+        ret and self.write_pg_hba()
+        return ret
```
Comment (a1exsh) [inline helpers/postgresql.py]: Doesn't look like a clear win to me. ;-)  What if you need to also "tune_pg_conf()" or something?  You'll get back to the old code anyway.

**Pull request merged**